src/app/portal/features/home-page/home-page.component.html

```diff
@@ -1,78 +1,86 @@
+  </h1>
+  <hr class="mb-7" />
+  <p class="text-justify">
+    The GHGA Data Portal is a secure national infrastructure for human omics data
+    available under controlled access. Access to the archived data can be requested from
+    the data controllers who are responsible for evaluating the data access request.
+  </p>
+  <p class="text-justify">
+    The datasets within are annotated following the
+    <a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://docs.ghga.de/metadata/overview/"
+      >GHGA Metadata Model<mat-icon>open_in_new</mat-icon></a
+    >, which is compatible with the metadata model of the EGA.
+  </p>
+  <p class="text-justify">
+    For further documentation, please visit the
+    <a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://docs.ghga.de/"
+      >GHGA User Documentation<mat-icon>open_in_new</mat-icon></a
+    >.
+  </p>
+  <div class="mb-8 w-full text-center">
+    <a class="text-tertiary underline" mat-raised-button routerLink="/browse"
+      >Browse data</a
+    >
+  </div>
+</section>
+<app-home-page-global-stats />
+<section>
+  <p class="text-justify">
+    GHGA is a national infrastructure to enable the FAIR and secure sharing of genetic
+    and other human omics data. It is embedded in European activities such as the
+    federated European Genome-Phenome Archive (<a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://ega-archive.org/about/projects-and-funders/federated-ega/"
+      >FEGA<mat-icon>open_in_new</mat-icon></a
+    >) and the European Genomic Data Infrastructure (<a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://gdi.onemilliongenomes.eu/"
+      >GDI<mat-icon>open_in_new</mat-icon></a
+    >).
+  </p>
+  <p class="text-justify">
+    GHGA is funded by the Deutsche Forschungsgemeinschaft (DFG, German Research
+    Foundation, Grant Number
+    <a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://gepris.dfg.de/gepris/projekt/441914366?context=projekt&task=showDetail&id=441914366&"
+      >441914366<mat-icon>open_in_new</mat-icon></a
+    >) as part of the National Research Data Infrastructure initiative (<a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://www.nfdi.de/?lang=en"
+      >NFDI<mat-icon>open_in_new</mat-icon></a
+    >) and by the contributing institutions.
+  </p>
+  <p class="text-justify">
+    More at
+    <a
+      class="text-tertiary underline"
+      target="_blank"
+      rel="noreferrer noopener"
+      href="https://www.ghga.de/"
+      >www.ghga.de<mat-icon>open_in_new</mat-icon></a
+    >.
+  </p>
+</section>
```

Comment:
The `<pre>` tag below can now be removed.

Reply:
Done 👍